.github/workflows/publish_nuget_package.yml

```diff
@@ -13,33 +13,30 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Setup .NET
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v4
         with:
           dotnet-version: '8.x'
 
-      - name: Get Latest NuGet Version
-        id: get_version
+      - name: Fetch The Tag
         run: |
-            VERSION=$(curl -sL "https://api.nuget.org/v3/registration5-gz-semver2/entrustsignhostclientlibrary/index.json" | gunzip -c | jq -r '.items[].items[].catalogEntry.version' | grep '^3' | sort -V | tail -n 1)
-            echo "LATEST_VERSION=${VERSION}" >> $GITHUB_ENV
-            echo "Latest version: $VERSION"
-            IFS='.' read -r -a VERSION_PARTS <<< "$VERSION"
+          # Fetch all tags from the remote repository
+          git fetch https://github.com/Evidos/SignhostClientLibrary.git --tags
```

Comment:
I recommend using `fetch-tags: true` in the checkout  action instead. That way everything is done in one step

Reply:
This works in the fork before it gets merged, and during this time, it fetches the tags from the fork, not from the origin. So if I do this, I need to add the same to both my fork's remote and the origin, which adds an extra step